events/dvg_lotharingia.txt

```diff
@@ -0,0 +1,1204 @@
+			}
+		}
+
+		if = {
+			limit = { exists = s:STATE_NOUVELLE_ALSACE.region_state:LOT }
+			create_country = { #Nouvelle Alsace
+				tag = NEA
+				origin = this
+				state = s:STATE_NOUVELLE_ALSACE.region_state:LOT
+				state = s:STATE_LIBREVILLE.region_state:LOT
+
+				on_created = {
+					warlord_setup_effect = yes
+					set_variable = warlord_state
+				}
+			}
+			c:NEA = { save_scope_as = scope_NEA}
+		}
+
+		if = {
+			limit = { exists = s:STATE_COTE_DU_CAFE.region_state:LOT }
+			create_country = { #Atlantia
+				tag = ATL 
+				origin = this
+				state = s:STATE_COTE_DU_CAFE.region_state:LOT
+				state = s:STATE_GOUAS.region_state:LOT
+
+				on_created = {
+					warlord_setup_effect = yes
+					set_variable = warlord_state
+				}
+			}
+			c:ATL = { save_scope_as = scope_ATL }
+			
+			
+		}
+
+		if = {
+			limit = { exists = s:STATE_MARCHES_OUEST.region_state:LOT }
+			create_country = { #Taragui
+				tag = TGI
+				origin = this
+				state = s:STATE_CHACO_BOREAL.region_state:LOT
+				state = s:STATE_MARCHES_OUEST.region_state:LOT
+				state = s:STATE_GRAND_CHACO.region_state:LOT
+				state = s:STATE_EOFERWIC.region_state:LOT
+				state = s:STATE_NORTHUMBRIA.region_state:LOT
+
+				on_created = {
+					warlord_setup_effect = yes
+					set_variable = warlord_state
+				}
+			}
+			c:TGI = { save_scope_as = scope_TGI }
+		}
+
+		
+
+		every_subject_or_below = {
+			make_independent = yes
+		}
+
+		c:ODO = {
+		 if = {
+			limit = {
+				has_variable = warlord_state
+				NOR = {
+					has_variable = var_warlord_two
+					has_variable = var_warlord_three
+					has_variable = var_warlord_four 
+				}
+			}
+			}
+			save_scope_as = scope_random_warlord_one
+			set_variable = var_warlord_one
+		}
+
+		c:NEA = {
+		if = {
+			limit = {
+				has_variable = warlord_state
+				NOR = {
+					has_variable = var_warlord_one
+					has_variable = var_warlord_three
+					has_variable = var_warlord_four
+				}
+			}
+			}
+			save_scope_as = scope_random_warlord_two
+			set_variable = var_warlord_two
+		}
+
+		c:ATL = {
+		if = {
+			limit = {
+				has_variable = warlord_state
+				NOR = {
+					has_variable = var_warlord_one
+					has_variable = var_warlord_two
+					has_variable = var_warlord_four
+				}
+			}
+			}
+			save_scope_as = scope_random_warlord_three
+			set_variable = var_warlord_three
+		}
+
+		c:TGI = {
+		if = {
+			limit = {
+				has_variable = warlord_state
+				NOR = {
+					has_variable = var_warlord_one
+					has_variable = var_warlord_two
+					has_variable = var_warlord_three
+				}
+			}
+			}
+			save_scope_as = scope_random_warlord_four
+			set_variable = var_warlord_four
+		}
+	}
+
+	option = {
+		name = dvg_lotharingia.5.ab
+		trigger = {
+			exists = scope:scope_random_warlord_one
+			is_player = yes
+		}
+		c:LOT = {
+		create_diplomatic_play = {
+			name = guarani_independence
+		
+			target_country = c:TGI
+			
+			war = yes
+			
+			type = dp_annex_war

```

Comment:
I added a special diplomatic play for rebellions.
https://github.com/vic3-divergences/divergences/blob/6e543adcf147d2568dc8a236d0e21a83ba510919/common/diplomatic_plays/dvg_diplomatic_plays.txt
dp_rebellion, its on https://github.com/vic3-divergences/divergences/pull/193/files for now.
Maybe can add that in here instead of dp_annex_war. This resolves some weirdness.
* Iniater does not get infamy
* No negotiated peace, either one gets its goals

Reply:
Well maybe this can get implemented later after it gets merged with the main branch?